ajax/ruleaction.php

```diff
@@ -64,6 +64,7 @@
     if ($_POST['ruleactions_id'] > 0) {
         $already_used = false;
     } else { // New action
+        /** @var RuleAction $ra */
```

Comment:
Applying the PHPDoc changes made to `getItemForType` on `main` and adding a return/phpstan-return tag to `Rule::getActionClass` of `class-string<RuleAction>` would be a more correct fix.